src/app/[lang]/(protected)/create-new-password/passwordsInput.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState, MouseEvent, ChangeEvent } from "react";
+import Image from "next/image";
+import { DictionaryReturnTypes } from "../../dictionaries";
+import { isValidPassword } from "@/util/input-validation";
+
+interface PasswordInputProps {
+  placeholder: string;
+  hint: boolean;
+  setPassHandler: (pass: string) => void;
+  pass: string;
+  dict: Awaited<DictionaryReturnTypes["/en/signup"]>;
```

Comment:
You are using the wrong `dict` type 

```suggestion
  dict: Awaited<DictionaryReturnTypes["/en/create-new-password"]>;

```